server/server.js

```diff
@@ -16,10 +18,12 @@ App.use(json());
 App.use(express.static(join(__dirname, "../public")));
 App.use(expressPino);
 
-App.use("/", router);
 
 let server = null;
 
+App.use("/", serverApi());
+ApplyExecutableMiddleware(App);
```

Comment:
Like I mentioned, this should be using a promise or `async/await`.

So either 

```
ApplyExecutableMiddleware(App).then(() => {
  //success
}).catch((error) => {
  // oops something happened - perhaps we should crash
})
```

or 

```
try {
  await ApplyExecutableMiddleware(App)
} catch(error) {
  // oops
}
```

Note that it is not possible to use `async/await` in a top level script though, it has to be inside an `async function` so perhaps this file could be refactored to run inside a function.